articles/active-directory/authentication/howto-password-ban-bad-on-premises-deploy.md

```diff
@@ -406,6 +406,8 @@ The current setting can be queried using the `Get-AzureADPasswordProtectionProxy
 
 The `Get-AzureADPasswordProtectionProxy` cmdlet may be used to query the software version of all currently installed Azure AD Password Protection proxy servers in a forest.
 
+Note that the proxy service will only automatically upgrade to a newer version when there are major feature updates or critical security patches needed. Currently, the proxy service will automatically upgrade any version older than 1.2.125.0.
```

Comment:
>upgrade to a newer version when there are major feature updates or critical security patches needed. Currently, the proxy service will automatically upgrade any version older than 1.2.125.0. [](http://example.com/codeflow?start=52&length=190)

The only thing I am comfortable with using auto-update is for critical security patches. Once we validate that it works and we have support for rings/SDP, then I think we can say that auto-update is GA.